client/components/language-picker/constants.js

```diff
@@ -0,0 +1,307 @@
+/**
+ * @format
+ * @jest-environment jsdom
+ */
+
+/*
+ * List of territories that languages are grouped into (displayed as tabs in the UI).
+ * Source of this data is the CLDR database, namely the `node-cldr` NPM package.
+ *
+ * `subTerritories` properties map the territory names into numerical geographic region
+ * codes according to the UN M49 (https://unstats.un.org/unsd/methodology/m49/) standard
+ * that is used in the CLDR database data.
+ *
+ * To move languages between territories or to include a language in multiple territories
+ * change the 'territories' arrays in language data in 'config/_shared.json'.
+ *
+ * Some languages, e.g., Esperanto are not included in any territory -- these are
+ * discoverable only with search.
+ *
+ * `countries` is a list of countries in each region. It's used to map country code
+ * retrieved from geolocation service into the name of the tab selected by default.
+ */
+export const DEFAULT_TERRITORY = 'asia-pacific';
+export const TERRITORIES = [
+	{
+		id: 'africa-middle-east',
+		name: translate => translate( 'Africa and Middle East' ),
+		subTerritories: [ '145', '002' ],
+		countries: [
+			'AE',
+			'AM',
+			'AZ',
+			'BH',
+			'CY',
+			'GE',
+			'IL',
+			'IQ',
+			'JO',
+			'KW',
+			'LB',
+			'OM',
+			'PS',
+			'QA',
+			'SA',
+			'SY',
+			'TR',
+			'YE',
+			'DZ',
+			'EG',
+			'EH',
+			'LY',
+			'MA',
+			'SD',
+			'TN',
+			'EA',
+			'IC',
+			'BF',
+			'BJ',
+			'CI',
+			'CV',
+			'GH',
+			'GM',
+			'GN',
+			'GW',
+			'LR',
+			'ML',
+			'MR',
+			'NE',
+			'NG',
+			'SH',
+			'SL',
+			'SN',
+			'TG',
+			'AO',
+			'CD',
+			'CF',
+			'CG',
+			'CM',
+			'GA',
+			'GQ',
+			'ST',
+			'TD',
+			'BI',
+			'DJ',
+			'ER',
+			'ET',
+			'KE',
+			'KM',
+			'MG',
+			'MU',
+			'MW',
+			'MZ',
+			'RE',
+			'RW',
+			'SC',
+			'SO',
+			'SS',
+			'TZ',
+			'UG',
+			'YT',
+			'ZM',
+			'ZW',
+			'BW',
+			'LS',
+			'NA',
+			'SZ',
+			'ZA',
+		],
+	},
+	{
+		id: 'americas',
+		name: translate => translate( 'Americas' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **97** times:
`translate( 'America' )` *ES Score: 20*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).